tools/db_bench_tool.cc

```diff
@@ -3372,6 +3372,17 @@ void VerifyDBFromDB(std::string& truth_db_name) {
     options.compression_opts.max_dict_bytes = FLAGS_compression_max_dict_bytes;
     options.compression_opts.zstd_max_train_bytes =
         FLAGS_compression_zstd_max_train_bytes;
+    if (FLAGS_cache_size) {
+      cache_ = NewCache(FLAGS_cache_size);
```

Comment:
why do we need a new cache? I thought it is instantiated in the same way in the constructor.